### PR TITLE
📦 Add SPM support

### PR DESCRIPTION
**Pull request by timmikeljmonzo**
### What does this PR do?

*  Adds SPM support
* Removes cocoapods support

Tested integration with main project by removing cocoapods and importing by pointing to latest commit on this branch

### CODE REVIEW

File: Sources/Presentr/CoverHorizontalAnimation.swift [hunk 1]
```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import CoreGraphics
```
Comment (MCR19) [inline Sources/Presentr/CoverHorizontalAnimation.swift:10]: Nit: Should we order these imports alphabetically?

**Review by MCR19 — APPROVED** [review]
Looks great! Just a nit but not blocking!

File: Sources/Presentr/CoverHorizontalAnimation.swift (hunk 1 above)
Comment (timmikeljmonzo) [reply Sources/Presentr/CoverHorizontalAnimation.swift:10]: I guess we should, normally swift format does this automatically in the main app

Comment (timmikeljmonzo) [reply Sources/Presentr/CoverHorizontalAnimation.swift:10]: I will leave this for now if thats cool

**Pull request merged**